### PR TITLE
feat: add CLIENT_ID key to userdata

### PR DESCRIPTION
**Pull request by jmolnar-zscaler**
under development

### CODE REVIEW

File: CHANGELOG.md
Comment (eliassun) [inline CHANGELOG.md:1]: The zsec should have an option to input the client_id just as it collects the managed identity.

Comment (jmolnar-zscaler) [reply CHANGELOG.md:1]: zsec actually prompts for the managed identity name and resource group that is used to populate the managed identity child module. That's how i'm gathering the details and then outputting the client_id back to main.tf

**Review by eliassun — APPROVED** [review]

File: examples/base_1cc/main.tf [hunk 1]
```diff
@@ -111,6 +111,7 @@ locals {
 CC_URL=${var.cc_vm_prov_url}
 AZURE_VAULT_URL=${var.azure_vault_url}
 HTTP_PROBE_PORT=${var.http_probe_port}
+CLIENT_ID=${module.cc_identity.managed_identity_client_id}
```
Comment (eliassun) [inline examples/base_1cc/main.tf:114]: As the new comments in the Janus's PR, we want to change the client_id in userdata.cfg to "azure_managed_identity_client_id"

Comment (jmolnar-zscaler) [reply examples/base_1cc/main.tf]: @eliassun fixed in latest commit

**Pull request merged**